docs/reference/modules/discovery/discovery-settings.asciidoc

```diff
@@ -227,3 +227,22 @@ APIs are not be blocked and can run on any available node.
 ===============================
 
 --
+
+`monitor.fs.health.enabled`::
+
+    (<<cluster-update-settings,Dynamic>>, boolean) Sets whether the node runs
```

Comment:
Nit: We try to use a "If `true`, ..." convention for boolean params. This is fine as-is though.
```suggestion
    (<<cluster-update-settings,Dynamic>>, boolean) If `true`, the node runs
```